src/ffi/mod.rs

```diff
@@ -75,6 +75,56 @@ pub use self::weakrefobject::*;
 #[cfg(not(Py_LIMITED_API))]
 pub use self::cpython::*;
 
+// skipped abstract.h
+// skipped asdl.h
+// skipped ast.h
+mod bltinmodule;
+mod boolobject; // TODO supports PEP-384 only; needs adjustment for Python 3.3 and 3.5
+mod bytearrayobject;
+mod bytesobject;
+// skipped cellobject.h
+mod ceval; // TODO supports PEP-384 only; needs adjustment for Python 3.3 and 3.5
+
+// skipped classobject.h
+mod codecs; // TODO supports PEP-384 only; needs adjustment for Python 3.3 and 3.5
+#[cfg(Py_LIMITED_API)]
+mod code {}
+#[cfg(not(Py_LIMITED_API))]
+mod code;
```

Comment:
See the above comment.
```suggestion
mod code {}
```